mixer/pkg/runtime/config/ephemeral.go

```diff
@@ -234,31 +236,49 @@ func (e *Ephemeral) processInstanceConfigs(counters Counters) map[string]*Instan
 	return instances
 }
 
-func (e *Ephemeral) processAdapterInfoConfigs(counters Counters) *adapterInfoRegistry {
-
+func (e *Ephemeral) processAdapterInfoConfigs(availableTmpls map[string]*Template, counters Counters) map[string]*Adapter {
+	result := map[string]*Adapter{}
 	log.Debug("Begin processing adapter info configurations.")
-
-	var adapterInfos []*v1beta1.Info
-
 	for adapterInfoKey, resource := range e.entries {
 		if adapterInfoKey.Kind != AdapterKind {
 			continue
 		}
+
+		adapterName := adapterInfoKey.String()
+
 		counters.adapterInfoConfig.Add(1)
 		cfg := resource.Spec.(*v1beta1.Info)
-		log.Debugf("Processing incoming adapter info: name='%s'\n%s", adapterInfoKey.String(), cfg)
-		adapterInfos = append(adapterInfos, cfg)
-	}
 
-	log.Debugf("Total received adapter info: count=%d, value='%v'", len(adapterInfos), adapterInfos)
-	reg, err := newAdapterInfoRegistry(adapterInfos)
-	if err != nil {
-		log.Errorf("Error when reading adapter info='%v'", err)
-		counters.adapterInfoConfigError.Inc()
+		log.Debugf("Processing incoming adapter info: name='%s'\n%v", adapterName, cfg)
```

Comment:
Should we introduce a logging scope for adapter config activity

Reply:
You mean the zoning stuff ? Is that functionality in ? IMO, the entire config ingestion related errors can be in its own zone (or logging scope) and we don't need special zoning just for adapter config activities.